### PR TITLE
Extract e2e test flow for recovery nag skipping

### PR DESCRIPTION
**Pull request by frederikrothenberger**
This PR extracts a flow for recovery nag skipping in the e2e tests, thus simplifying the code.

### CODE REVIEW

**Review by lmuntaner — APPROVED** [review]
LGTM!

What does "Nag" mean?

File: src/frontend/src/test-e2e/flows.ts [hunk 1]
```diff
@@ -92,12 +92,8 @@ export const FLOWS = {
     await welcomeView.login();
     await welcomeView.typeUserNumber(userNumber);
     await browser.$("button[data-action='continue']").click();
-    // NOTE: depending on the browser, we issue different warnings. On Safari,
-    // the warning comes before the recovery method selector. Since we only
-    // test on Chrome we always expect the recovery selector first.
-    const recoveryMethodSelectorView = new RecoveryMethodSelectorView(browser);
-    await recoveryMethodSelectorView.waitForDisplay();
-    await recoveryMethodSelectorView.skipRecovery();
+    // NOTE: handle recovery nag because there is no recovery phrase
```
Comment (lmuntaner) [inline src/frontend/src/test-e2e/flows.ts:95]: Is this a TODO or a note?

Comment (frederikrothenberger) [reply src/frontend/src/test-e2e/flows.ts:95]: this is a note.

**Comment by frederikrothenberger** [comment]
> What does "Nag" mean?

The recovery nag (screen) is a screen nagging the user to create a recovery phrase.

File: src/frontend/src/test-e2e/flows.ts (hunk 1 above)
Comment (lmuntaner) [reply src/frontend/src/test-e2e/flows.ts:95]: Sorry, I don't understand the note then.

What is it for?

Comment (frederikrothenberger) [reply src/frontend/src/test-e2e/flows.ts:95]: See https://github.com/dfinity/internet-identity/pull/1928.

**Pull request merged**